backend/src/main/resources/templates/uploadPhoto.html

```diff
@@ -120,6 +121,15 @@ <h5 th:text="${permission}">PERMISSION_NAME</h5>
         document.body.removeChild(el);
     }
 </script>
-
+<template id="name-input-template">
+    <div class="field-group">
+        <label class="name-label">Cím a galériában
+            <span class="file-name"></span>
+        </label>
+        <div style="clear: both"></div>
+        <input type="text" class="name-input" name="names" autocomplete="off" />
+        <small>Vessző ne legyen a névben</small>
```

Comment:
I'd just remove them from the name if there is any. Maybe mention that special chars will be removed.

Reply:
I take the file names as a list of strings and spring fails to parse it if there are commas inside